lib/processaws/aws.go

```diff
@@ -0,0 +1,129 @@
+package processaws
```

Comment:
Maybe we can talk tomorrow about why we needed our own copy of this class instead of using the existing one? I'd like to understand so we can outline this well for Craig.

Reply:
A separate one is not required for aws. I was not sure about a method `isGarbageCollectAccount` hence created a separate module for aws. Removing it.

Reply:
There is one more place where the code is different from gcp, `ProcessActiveWork`